src/main/java/com/sayweb/controller/AppController.java

```diff
@@ -114,17 +117,16 @@ public ResponseEntity<InputStreamResource> getAudioFile(@PathVariable String fil
 
   private void cleanupOldFiles(File audioDir) {
     File[] files = audioDir.listFiles();
-    if (files != null) {
-      for (File file : files) {
-        if (file.isFile() && isOlderThanOneDay(file)) {
-          file.delete();
-        }
+    if (files == null) return;
```

Comment:
All statements should be separated from each other by a blank line. Otherwise, it is difficult to distinguish them from each other.

No:
```
if ...
for ...
```
 Yes:
```
if ... 

for ...
```